web/app/src/ui/games-list/GameCard.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Game, getGameGreenPlayer, getGameRedPlayer} from "../../model/game";
+import {Link} from "react-router-dom";
+
+/**
+ * Game card component.
+ */
+export function GameCard({game}: {
+	game: Game;
```

Comment:
GitHub shows different indentation for this file. Is it an artifact? If not, you should definitely use an automated code formatter like prettier.